src/Native/System.Native/pal_networking.cpp

```diff
@@ -13,6 +13,19 @@
 #include <sys/socket.h>
 #include <unistd.h>
 #include <vector>
+#include <errno.h>
+
+#if defined(__APPLE__)
```

Comment:
The check is not required, it's simply being explicit about the target OS. I wondered if the BSDs would also be missing these definitions.